x-pack/solutions/security/plugins/security_solution/public/asset_inventory/pages/index.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { SecuritySolutionPageWrapper } from '../../common/components/page_wrapper';
+import { useKibana } from '../../common/lib/kibana';
+import { SecurityPageName } from '../../../common/constants';
+import { SpyRoute } from '../../common/utils/route/spy_routes';
+
+export const AssetInventoryContainer = React.memo(() => {
+  const { assetInventory } = useKibana().services;
+
+  return (
+    <SecuritySolutionPageWrapper noPadding>
+      {assetInventory.getAssetInventoryPage({})}
```

Comment:
Would it make more sense to make it just ```{assetInventory.getAssetInventoryPage()}``` here?

Are we able to assembled the param `AppPluginStartDependencies` the function needs in x-pack/solutions/security/plugins/asset_inventory/public/plugin.ts?

Reply:
I agree with you, though this is just temporary because we are not passing in any dependency for now